homeworks/sasha.demidenko_SashaDemi/5-logic/calculator.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Basic JS</title>
+        <style>
+            .field {
+                display: flex;
+                column-gap: 20px;
+                margin-left: 60px;
+            }
+        </style>
+        <link rel="stylesheet" href="calculator.css">
+    </head>
+    <body class="body-main">
+        <header class="calculator_header--main">
+            <img src="../GamesProject/Images/about_logo.png" alt="no img" class="calculator_header--logo">
+            <nav class="calculator_header--nav">
+                <ul class="calculator_header--ul">
+                    <li class="calculator_header--li calculator_li--underline">Calculator</li>
```

Comment:
If these links don't lead anywhere, please remove them 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/815576aa-3c1d-4078-9a90-be98cf77af63)

Reply:
Or move content of`5-logic` folder to your project files